stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -543,8 +523,8 @@ func.func @select_and_scatter(
     window_strides = dense<[1, 2, 2, 1]> : tensor<4xi64>
   } : (tensor<10x24x24x64xf32>, tensor<10x12x12x64xf32>, tensor<f32>) ->
         tensor<10x24x24x64xf32>
+  // CHECK: %2 = "hlo_test_infer.return_types"(%1) {types0 = tensor<10x24x24x64xf32>} : (tensor<10x24x24x64xf32>) -> tensor<10x24x24x64xindex>
```

Comment:
Also can we simplify `{types0 = tensor<10x24x24x64xf32>}` to just `types0 = tensor<10x24x24x64xf32>`? That would make things easier to read and write.